### ISSUE TITLE
Class Member type hints aren't stripped

### ISSUE THREAD

**Issue by Zadagu** [abarker/strip-hints#2]
I can annotate class members and they wont stripped. example output:
```Python
class PlaneCalculation:
    def __init__(self, sim                    , plane         , reference_point            , step_size       , spectrum_samples       ):
        self.__sim: FrequencySimulator = sim
        self.__plane: 'Plane' = plane
        self.__reference_point = reference_point
        self.__step_size = step_size
        self.__calculated_spectrum = None
```

**Comment by abarker** [comment]
Technically, those are annotated expressions rather than annotated variables.  The README noted that annotated expressions are not handled by the program.  I went ahead and modified the code to handle that particular case of annotated expressions because it was fairly simple.  Try the new version I've uploaded.

General annotated expressions are not currently handled, but most cases would not be too hard to add.  There's an example file in the test directory that has some examples of general annotated expressions.
